### PR TITLE
Serenity updates - September - New features rules

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

File: content/updates/2020-09/1-be-more-and-more-productive-when-using-rules.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+::: meta-data type="New" features="Rules engine, Productivity" available="late September" link-to-doc="../articles/manage-your-rules.html#duplicate-a-rule"
+:::
+
+In order to avoid re-writing the whole rule, you are now able to duplicate it in two clicks! Indeed, sometimes, you may have very similar rules with only one condition that is different, for instance. What a time saver, isn't it? :wink:
+
+## Define a status for your rules
+::: meta-data type="New" features="Rules engine, Productivity" available="late September" link-to-doc="../articles/manage-your-rules.html#define-a-status-for-a-rule"
+:::
+
+This new status enables you to activate or deactivate your rules.
```
Comment (NolwennP) [inline content/updates/2020-09/1-be-more-and-more-productive-when-using-rules.md:20]: If you want to go to the next line, add a double space at the end of this one.

File: content/updates/2020-09/1-be-more-and-more-productive-when-using-rules.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+For example, if the creation/edition of the rule is not finished yet, setting its status to "disabled" will pause the automatic execution of the rule.
```
Comment (NolwennP) [inline content/updates/2020-09/1-be-more-and-more-productive-when-using-rules.md:21]: Idem for this line. ;)

**Review by NolwennP — COMMENTED** [review]
I would add a screenshot of the duplicate button and another one for the status. It's always better with screenshots. ;) If you don't have them now. Just mention it. You can add them in a different PR if this one is already merged.

File: content/updates/2020-09/1-be-more-and-more-productive-when-using-rules.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+In order to avoid re-writing the whole rule, you are now able to duplicate it in two clicks! Indeed, sometimes, you may have very similar rules with only one condition that is different, for instance. What a time saver, isn't it? :wink:
+
+![Duplicate your rule](../img/Rules_Duplicate.png)
+
+## Define a status for your rules
+::: meta-data type="New" features="Rules engine, Productivity" available="late September" link-to-doc="../articles/manage-your-rules.html#define-a-status-for-a-rule"
+:::
+
+This new status enables you to activate or deactivate your rules.
```
Comment (LaureBro) [inline content/updates/2020-09/1-be-more-and-more-productive-when-using-rules.md:24]:
```suggestion
We introduced a new property for rules: an enabled/disabled status. This new status allows you to easily activate or deactivate your rules.  
```

**Review by LaureBro — COMMENTED** [review]
2 comments ;) thanks!

**Review by LaureBro — APPROVED** [review]

**Pull request merged**